### DISCUSSION TITLE
Failed Experiments Handling

### DISCUSSION THREAD

**Discussion by Demydoff** [abrechts/Phoenix-ELN#19] [Feature Proposals]
Are you planning to add a function to mark an experiment as failed? For example, when you right-click now, the only option is 'Finalize Experiment,' but you could add an 'Finalize Failed Experiment' option. After that, the project folder could display the number of successful and failed experiments. Instead of a checkmark, a cross could be added to indicate failed experiments.

Do you consider this functionality redundant?
![image](https://github.com/user-attachments/assets/9b2a9c79-f5fd-4b3c-9de3-657a637b991d)
![image](https://github.com/user-attachments/assets/9669663b-dfd6-4938-939f-c0d4538faed8)

**Comment by abrechts** [comment]
The question is what is considered as a failed experiment. If it is an experiment without obtaining a product, then you can easily identify it by the "no yield" label, e.g. "demo-00006 - **no yield** (10 g)" in the experiment tree. Therefore, in this case, an extra icon and an additional finalize option indeed seem a bit redundant.

Concerning counting successful and failed experiments: Could you elaborate a bit more on this idea, since I honestly have difficulties to see the added value of such statistics from the scientific perspective.
